frontend/src/App.tsx

```diff
@@ -30,32 +31,34 @@ const App = () => (
     <AuthProvider>
         <LanguageProvider>
             <SignalRProvider>
-                <InstallationProvider>
-                    <MissionDefinitionsProvider>
-                        <RobotProvider>
-                            <MissionRunsProvider>
-                                <AlertProvider>
-                                    <SafeZoneProvider>
-                                        <MissionRunsProvider>
-                                            <MissionControlProvider>
-                                                <UnauthenticatedTemplate>
-                                                    <div className="sign-in-page">
-                                                        <AssetSelectionPage></AssetSelectionPage>
-                                                    </div>
-                                                </UnauthenticatedTemplate>
-                                                <AuthenticatedTemplate>
-                                                    <MissionFilterProvider>
-                                                        <FlotillaSite />
-                                                    </MissionFilterProvider>
-                                                </AuthenticatedTemplate>
-                                            </MissionControlProvider>
-                                        </MissionRunsProvider>
-                                    </SafeZoneProvider>
-                                </AlertProvider>
-                            </MissionRunsProvider>
-                        </RobotProvider>
-                    </MissionDefinitionsProvider>
-                </InstallationProvider>
+                <MediaStreamProvider>
```

Comment:
Consider moving inside the the installation provider or however far inside makes sense

Reply:
Have moved the provider further down the chain.